tests/check_localregistry.sh

```diff
@@ -0,0 +1,27 @@
+# !/bin/bash
+
+count=0
+echo "Waiting for registry proxy to be active..."
+until $(kubectl -n kube-system get pods | grep -q '^kube-registry-proxy.*1/1.*$'); do
```

Comment:
At least for circleci the pattern is to start all components, then wait for them in a separate step.
So we don't serialize the wait. 

Also I don't see why this needs to be setup on each test.

Reply:
But please confirm that this will not run in circleci or local testing - since neither requires a registry.

Reply:
Actually, we will want CI to use this.  The idea for the in-cluster registry is that we push images for the local code to the registry when the framework is started in order to ensure that we're running with the current code.  If we have this, we won't need CI to use an external docker registry at all.